packages.md

```diff
@@ -0,0 +1,10 @@
+# Packages
+
+This page contains packages recommended by HelloFresh Golang Clan. These are not just the cool repos around, but the packages that we found really practical and worth to use over the others.
+
+If you need more packages or just for inspiration, visit [Awesome Go](https://awesome-go.com/).
+
+## Database
```

Comment:
I think we could add https://upper.io/db.v3/ to this list @bastiankoetsier WDYT?

Reply:
@trustmaster is right. Also, I did not test enough of this lib to really recommend it.